mythtv/settings.pro

```diff
@@ -186,6 +186,8 @@ win32 {
             QMAKE_EXTENSION_LIB=a
         }
         MYTH_LIB_EXT  =$${LIBVERSION}.$${QMAKE_EXTENSION_LIB}
+        
+        DEFINES += MUI_API
```

Comment:
I was able to pull your tree  and get it to compile. The problem here is that the templated function in mythdialogbox.h
https://github.com/MythTV/mythtv/blob/a2cdb9047a68d0d36c8be1493b68a88f6dfe6af0/mythtv/libs/libmythui/mythdialogbox.h#L497
shouldn't have the MUI_PUBLIC marking. Removing that marking allows your tree to compile.

I've committed this change to mythdialogbox.h, so you should be able to update your tree from git and remove this change, and it should compile.

Reply:
Thanks. I have removed the MUI_PUBLIC marking and have recompiled without any issues.